### DISCUSSION TITLE
Windows 7 support please 🙏🏻

### DISCUSSION THREAD

**Discussion by linchoxxx** [nefarius/HidHide#89] [Ideas]
Is there a possibility that you could make a w7 support? Its a great tool, but my old arcade cant use HidHide. Please make a W7 variation on the future. Thanks.

**Comment by nefarius** [comment]
You can't sign drivers for Windows 7 without WHQL procedure anymore thanks to Microsoft restrictions, so no, not worth it I'm afraid.

**Reply by linchoxxx** [reply]
I understand, it's sad, no way, I thought you were more pro programers. Thanks for the answer 🙏🏻

**Reply by nefarius** [reply]
I don't think you even remotely understood what I wrote but it's OK.

**Reply by linchoxxx** [reply]
Sorry if I offended you, I didn't mean to (but apparently I touched meat). I hope to find the solution for windows 7 with some other software, good luck. Thanks.

**Reply by nefarius** [reply]
I'm not offended but this machine translated jibberish doesn't make a whole lot of sense.
